normandy/recipes/tests/test_serializers.py

```diff
@@ -15,6 +15,7 @@ def test_it_works(self, rf):
         assert serializer.data == {
             'name': recipe.name,
             'id': recipe.id,
+            'last_updated': Whatever(),
```

Comment:
Can you do something like `recipe.last_updated`? I guess that would still have the type error. Maybe convert it to a string with `recipe.last_updated.isoformat()`?